File: easy-solution/Antidote21/905.java
class Solution {
    public int[] sortArrayByParity(int[] nums) {
        int even_num,length=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]%2==0){
                even_num = nums[i];
                nums[i] = nums[length];
                nums[length] = even_num;              
                length++;
            }
        }
        return nums;
    }
} 

File: easy-solution/pandora2948/905.py
class Solution:
    def sortArrayByParity(self, nums: List[int]) -> List[int]:
      oddNum = sorted(filter(lambda ele : ele % 2 != 0, nums))
      evenNum = sorted(filter(lambda ele : ele % 2 == 0, nums))
      

      nums.clear()
      
      for i in oddNum :
        evenNum.append(i)
      
      return evenNum
      
      
class Solution:
  def sortArrayByParity(self, nums: List[int]) -> List[int]:
    left = 0
    right = len(nums) - 1
  
    while left < right :
      if nums[left] % 2 == 0 :
        left += 1
        continue
      
      else :
        nums[left], nums[right] = nums[right], nums[left]
      right -= 1
    return nums